src/components/TextInput/TextInput.js

```diff
@@ -575,26 +558,38 @@ class TextInput extends React.Component<Props, State> {
                   {!!leftSideText && <AddonRegularText>{leftSideText}</AddonRegularText>}
                 </LeftSideWrapper>
               </TouchableWithoutFeedback>}
-              {!fullWidthSelector &&
-              <InputField
-                {...inputProps}
-                innerRef={(input) => {
-                  if (getInputRef) getInputRef(input);
-                  this.multilineInputField = input;
-                }}
-                onChange={this.handleChange}
-                onBlur={this.handleBlur}
-                onEndEditing={this.handleBlur}
-                onFocus={this.handleFocus}
-                onSubmitEditing={this.handleSubmit}
-                value={textInputValue}
-                autoCorrect={autoCorrect}
-                style={[defaultInputStyle, customStyle, additionalStyle, !editable && { color: colors.accent }]}
-                onLayout={onLayout}
-                placeholderTextColor={colors.accent}
-                alignTextOnRight={!!numeric}
-                smallPadding={!!onRightAddonPress}
-              />}
+              {!fullWidthSelector && (
+                <TouchableWithoutFeedback style={{ flex: 1 }} onPress={this.focusMultilineInput}>
+                  <View style={{ flex: 1 }}>
+                    <Tooltip
+                      body={errorMessage || ''}
+                      isVisible={!!hasError}
+                      wrapperStyle={{ alignSelf: 'flex-end' }}
+                    >
+                      <InputField
+                        {...inputProps}
+                        innerRef={(input) => {
+                        if (getInputRef) getInputRef(input);
+                          this.multilineInputField = input;
+                        }}
+                        onChange={this.handleChange}
+                        onBlur={this.handleBlur}
+                        onEndEditing={this.handleBlur}
+                        onFocus={this.handleFocus}
+                        onSubmitEditing={this.handleSubmit}
+                        value={textInputValue}
+                        autoCorrect={autoCorrect}
+                        style={[defaultInputStyle, customStyle, additionalStyle, !editable && { color: colors.accent }]}
```

Comment:
Input is not taking the whole `ItemHolder` width, so even regular (not numeric) inputs have text aligned to the right ;/